content/en/docs/tasks/configure-pod-container/assign-resources/allocate-devices-dra.md

```diff
@@ -0,0 +1,310 @@
+{{< glossary_tooltip text="ResourceClaims" term_id="resourceclaim" >}}. You can
+directly create a ResourceClaim or use a
+{{< glossary_tooltip text="ResourceClaimTemplate" term_id="resourceclaimtemplate" >}}
+to let Kubernetes generate and manage per-Pod ResourceClaims. 
+
+### Create a ResourceClaim {#create-resourceclaim}
+
+Manually create ResourceClaims if you want multiple Pods to share access to the
+same devices, or if you want a claim to exist beyond the lifetime of a Pod.
+If you directly reference a specific ResourceClaim in a Pod, that ResourceClaim
+must already exist in the cluster. If a referenced ResourceClaim doesn't exist,
+the Pod remains in a pending state until the ResourceClaim is created. 
+
+You can reference an auto-generated ResourceClaim in a Pod, but this isn't
+recommended because auto-generated ResourceClaims are bound to the lifetime of
+the Pod that triggered the generation.
+
+1. Save the following ResourceClaim as `basic-dra-resourceclaim.yaml`:
+
+   ```yaml
+   apiVersion: resource.k8s.io/v1beta2
+   kind: ResourceClaim
+   metadata:
+     name: big-blue-claim
+   spec:
+     devices:
+       constraints:
+       - matchAttribute: "resource-driver.example.com/color"
+       requests:
+       - name: big-blue-request
+         deviceClassName: example-device-class
+         allocationMode: All
+         selectors:
+         - cel:
+            expression: |-
+              device.attributes["resource-driver.example.com"].color == "blue" &&
+              device.attributes["resource-driver.example.com"].size == "big"
+   ```
+   This ResourceClaim requests devices in the `example-device-class` DeviceClass
+   that match both of the following parameters:
+
+   * `resource-driver.example.com/color` attribute with a value of `blue`
+   * `resource-driver.example.com/size` attribute with a value of `big`
+
+   The `constraints` field indicates that only devices that have an attribute
+   named `resource-driver.example.com/color` can be allocated to this
+   ResourceClaim.
+
+1. Create the ResourceClaim: 
+
+   ```shell
+   kubectl apply -f basic-dra-resourceclaim.yaml
+   ```
+
+For every available field, see the
+{{< api-reference page="workload-resources/resource-claim-v1beta2" text="ResourceClaim reference" >}}.
+
+### Create a ResourceClaimTemplate {#create-resourceclaimtemplate}
+
+Create a ResourceClaimTemplate if you want every Pod to have access to separate
+devices that have similar configurations. For example, you might want simultaneous
+access to devices for Pods in a Job that uses
+[parallel execution](/docs/concepts/workloads/controllers/job/#parallel-jobs).
+
+1. Save the following ResourceClaimTemplate as
+   `basic-dra-resourceclaimtemplate.yaml`:
+
+   ```yaml
+   apiVersion: resource.k8s.io/v1beta2
+   kind: ResourceClaimTemplate
+   metadata:
+     name: large-black-cat-claim-template
+   spec:
+     spec:
+       devices:
+         constraints:
```

Comment:
Since this claim is only requesting one device, these constraints don't have any effect AFAIK. Is it worth simplifying this example by removing it?